### ISSUE TITLE
Linux Mint LMDE (Debian) version Installer Fails 

### ISSUE THREAD

**Issue by justsml** [docker/docker#16498]
The installer does not support  Linux Mint Debian Edition LMDE 2.
The returned error message is:
`Either your platform is not easily detectable, is not supported by this installer script...`

```sh
# uname -a
.... 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt11-1+deb8u4 (2015-09-19) x86_64 GNU/Linux

# lsb_release -a
Distributor ID:	LinuxMint
Description:	LMDE 2 Betsy
Release:	2
Codename:	betsy

```

PR attached

**Comment by justsml** [comment]
```sh
uname -a:
.... 3.16.0-4-amd64 #1 SMP Debian 3.16.7-ckt11-1+deb8u4 (2015-09-19) x86_64 GNU/Linux

```

**Comment by ddurdle** [comment]
There is an issue with LXC on LMDE2 (LXC breaks the OS).  Probably something related to queasy state systemd.  LXC and docker works under LMDE UP8 just fine.

**Comment by justsml** [comment]
@ddurdle The only issue I ran into on LMDE2 was it only liked running AUFS, not devicemapper but that might be related to my testing many different storage technologies: ZFS, LVM2 (w/ gratuitous experimentation + primary/mirroring), EXT3 & 4, GlusterFS, NFS, iSCSI - I know, I'm clearly trying to confuse myself. 

LXC seems to run quite well. 
**Is there some specific command I could try to repro your experience?**

**Comment by ddurdle** [comment]
After installing, when you try to runs a docker, it complains it can't remove some cgroups.  Seems like it is already using cgroups so it is conflicting.  Like I said, LXC used to work but after upgrading to LMDE2, same issue with LXC.  Tried on multiple LMDE2 systems.

**Comment by justsml** [comment]
I just realized I only tried on fresh LMDE2 installs! Could the difference lie there? 

Does cgroups need modern CPU features? Or specific matching kernel version + matched modules? 

A while back I had to enable some CPU features via KVM/QEMU cmd line options before my test docker env would stay running, it also had a dead network - though I can't remember if the CPU opts alone fixed all of it.

I'm still working on getting a better understanding of each sub-component used by docker and how they fit into the larger Linux ecosystem.
